LevelEditor/main.lua

```diff
@@ -54,6 +60,13 @@ function love.keypressed(key)
 		x, y = Screen.adjustPositionForScreen(x, y)
 		Screen.setExit(x, y)
 	end
+	if key == "s" and love.keyboard.isDown("lctrl") then
+		SaveFile.save(Screen.getAllTiles(), "../level.lua")
```

Comment:
if you run the program any way other than just from inside the level editor directory this will save to a randomish spot
id say it should look in 1 or 2 parent directories for a `levels` directory and save it there, and if a directory does not exist then save it to the current directory